scrapper.py

```diff
@@ -41,21 +44,38 @@ def get_search_urls(self):
         """
         Returns seed_urls param
         """
-        pass
+        return self.seed_urls
 
 
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
+        os.rmdir(base_path)
+    except FileNotFoundError:
+        pass
+    os.mkdir(base_path)
 
 
 def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path, 'r', encoding='utf-8') as file:
+        config = json.load(file)
+    seed_urls = config['seed_urls']
+    max_articles = config['total_articles_to_parse']
+    if not isinstance(max_articles, int) or max_articles <= 0:
+        raise IncorrectNumberOfArticlesError
+    if not isinstance(seed_urls, list) or not seed_urls:
+        raise IncorrectURLError
+    if max_articles > 100:
```

Comment:
I recommend taking a bigger number as 100 is a required number of articles to find and parse, so it is a bit odd that this code is expected to work for 100 items but is going to complain about 101